how2meet/main.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python3
+from how2meet import frontend
+from fastapi import FastAPI
+
+app = FastAPI()
+
+
+@app.get('/')
+def read_root():
+    return {'Hello': 'World'}
+
+
+frontend.init(app)
+
+if __name__ == '__main__':
+    print('Please start the app with the "uvicorn" command as shown in the start.sh script')
```

Comment:
where is the the `start.sh` script? I assume that's where the makefile comes into play?